SportsHub/UnitTests/MockData/UsersMockData.cs

```diff
@@ -0,0 +1,37 @@
+﻿using SportsHub.Domain.Models;
+
+namespace UnitTests.MockData;
+
+public class UsersMockData
```

Comment:
No need to create a separate class to return a collection of Users. You can move this method into `UserMockData` class.

Reply:
I am agree with Mykola, there are too much MockData classes